File: spec/code_samples/Node unirest/exchange-keys@{keyId}/parameters.js
var unirest = require("unirest");

var req = unirest("PARAMETERS", "http://ec2-3-222-208-106.compute-1.amazonaws.com:8086/exchange-keys/%7BkeyId%7D");


req.end(function (res) {
  if (res.error) throw new Error(res.error);

  console.log(res.body);
});

File: spec/code_samples/Python python3/exchange-keys@{keyId}@activate/parameters.py
import http.client

conn = http.client.HTTPConnection("ec2-3-222-208-106.compute-1.amazonaws.com:8086")

conn.request("PARAMETERS", "/exchange-keys/%7BkeyId%7D/activate")

res = conn.getresponse()
data = res.read()

print(data.decode("utf-8"))